src/lib/viewers/archive/__tests__/Breadcrumbs-test-react.js

```diff
@@ -20,52 +20,19 @@ describe('lib/viewers/archive/Breadcrumbs', () => {
 
     describe('render()', () => {
         test('should render correct components', () => {
-            const component = getComponent({ filename, fullPath, onClick, view });
+            renderView({ filename, fullPath, onClick, view });
 
-            expect(component.find('.bp-Breadcrumbs').length).toBe(1);
-            expect(component.find('Breadcrumb').length).toBe(1);
-            expect(component.find('PlainButton').length).toBe(3);
+            expect(screen.getByLabelText('Breadcrumb')).toBeInTheDocument();
+            expect(screen.getAllByRole('button')).toHaveLength(3);
+            expect(screen.getAllByRole('button').at(0).textContent).toContain('test.zip');
+            expect(screen.getAllByRole('button').at(1).textContent).toContain('test');
```

Comment:
Nit: you could do `screen.getAllByRole('button')` only once:
```suggestion
            const buttons = screen.getAllByRole('button')
            expect(buttons.at(1).textContent).toContain('test');
```